doc/user-manual/tools/auto.rst

```diff
@@ -0,0 +1,8 @@
+.. _auto:
+
+****
+Auto
```

Comment:
I think the title "Auto" is not very descriptive. Maybe something along the lines of "Automatic Proof Search (Auto)" is better?